src/pages/Home/Home.tsx

```diff
@@ -10,33 +13,37 @@ export function Home() {
   }
 
   return (
-    <div>
-      <h1>Home</h1>
-      <p>Welcome to the Tamriel Companion App!</p>
-      <div>
+    <Container size="sm" className={classes.container}>
+      <Title ta="center" mb="10">
+        Welcome to Tamriel Companion App!
+      </Title>
+      <Box className={classes.box}>
         <Button
           variant="filled"
           color="cyan"
           size="md"
           radius="md"
-          style={{ marginBottom: '10px' }}
+          className={classes.buttons}
+          leftSection={<Avatar src={antiquities} alt="Antiquities" radius="sm" />}
           onClick={() => handleNavigate('/antiquities')}
```

Comment:
consider using a sort of the constant for routes. Currently if you refactor Router.tsx, this will be not changed.